nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -1233,9 +1225,6 @@ in
             serviceConfig.LoadCredential = [
               "adminpass:${cfg.config.adminpassFile}"
             ] ++ runtimeSystemdCredentials;
-            # On Nextcloud ≥ 26, it is not necessary to patch the database files to prevent
-            # an automatic creation of the database user.
-            environment.NC_setup_create_db_user = lib.mkIf (nextcloudGreaterOrEqualThan "26") "false";
```

Comment:
I think this is still relevant, no?
Not the `mkIf`, but the declaration.

AFAIK it got added to the codebase and kept, you just need a very special (IMHO suboptimal) way of configuring Nextcloud to trigger it which is IIRC not actively tested.

Reply:
Ah yes, I read the check the wrong way around. It defaults to true, but we always want to have it false: https://github.com/nextcloud/server/blob/9cc2b01d5f43605837a9af72f55b989066d8cfe7/lib/private/Setup/AbstractDatabase.php#L73.
Even before it would have been fine to always set the env var as older versions would simply not make sense of it.